defaults/main.yml

```diff
@@ -14,3 +14,5 @@ r_packages_repos: "{{ r_cran_mirror }}"
 r_packages_lib: /usr/local/lib/R/site-library
 r_packages: []
 r_environment: {}
+
+r_scripts_install_path: /usr/local/bin
```

Comment:
Maybe also introduce: `r_scripts_install_owner` and `r_scripts_install_group`?

Reply:
Default `root`

Reply:
Yes I have think about it.

I think it is not needed, as ansible will use the running user as default for owner and group when templating.

And I does not know how to set this variable with a smart value, as the var 'ansible_user' may not be reliable:
see : https://github.com/ansible/ansible/issues/23530

Reply:
It will make it more difficult for us as we while have to find a clean way to find the current user running the role.

But if you want those var before merge. Please confirm. And I will add them.

Reply:
I think I prefer adding them. I think we use `ansible_user_id`. Does that suit not needs?

Reply:
done, see: https://github.com/Oefenweb/ansible-r/pull/52/commits/f9779dfe39caf0245dcbb23ada8e009f7aad0d3e